scripts/take_bias.sh

```diff
@@ -30,15 +30,15 @@ P=$1
 SAVE_FILE=${2:-"bias-%Y%m%d-%H%M%S.cr2"}
 echo "Taking bias frames on port ${P}: ${SAVE_FILE}"
 
-NUM_FRAMES=${4:-1}
-SHUTTER_INDEX=${3:-52} # 1/4000s on EOS 100D
+NUM_FRAMES=${3:-1}
+SHUTTER_INDEX=${4:-52} # 1/4000s on EOS 100D
 
 # Set to fast speed
 gphoto2 --port=${P} --set-config-index shutterspeed=${SHUTTER_INDEX}
 
 COUNTER=0
 until [[ ${COUNTER} -eq ${NUM_FRAMES} ]]; do
-    gphoto2 --port=${P} --filename=${SAVE_FILE} --capture-image-and-download
+    gphoto2 --port=${P} --filename=${SAVE_FILE}_${COUNTER} --capture-image-and-download
```

Comment:
Hmm. This seems problematic because the _0, _1, etc. will appear after the file extension (e.g. cr2).
I'd be inclined to add the count to the name only if NUM_FRAMES is > 1, and then to insert it before the file name extension.

Reply:
Good point. I didn't think that through. It's easy enough to do a shell substitution (i.e. `${SAVE_FILE/\.cr2/_${COUNTER}\.cr2}` however I suspect the real issue here is the output directory, not really the filename. The default name has a timestamp so would be okay for any NUM_FRAME. What about just having an OUTPUT_DIR argument and keep the default filename?

Reply:
Actually, maybe I don't like that idea. I'll probably just add the counter if NUM_FRAMES > 1 and SAVE_FILE is not empty.